src/component/list.js

```diff
@@ -45,26 +45,16 @@ ListContent.propTypes = {
 //
 
 export class List extends Component {
-  constructor(props) {
-    super(props);
-    this.ds = new ListView.DataSource({
-      rowHasChanged: (r1, r2) => r1 !== r2,
-    });
-  }
-
   get dataSource() {
-    return this.ds.cloneWithRows(this.props.data);
+    return this.props.data;
   }
 
   render() {
     return (
-      <ListView
-        ref={component => (this.list = component)}
-        dataSource={this.dataSource}
-        renderHeader={this.props.renderHeader}
-        stickyHeaderIndices={this.props.renderHeader ? [0] : []}
-        renderRow={this.props.renderItem}
-        enableEmptySections={true}
+      <FlatList
```

Comment:
I think the list headers are no longer sticky. Minor regression

Reply:
Nice catch. Not a showstopper imho since this PR resolves a major issue with empty lists not rendering. But feel free to create an issue if you feel we should fix this.